edweiss-app/app/(app)/lectures/remotecontrol/index.tsx

```diff
@@ -33,7 +33,7 @@ const RemoteControlScreen: ApplicationRoute = () => {
     const { courseNameString, lectureIdString } = useLocalSearchParams();
     const courseName = courseNameString as string;
     const lectureId = lectureIdString as string;
-    const [lectureDoc] = usePrefetchedDynamicDoc(CollectionOf<Lecture>(`courses/${courseName}/lectures`), lectureId, undefined);
+    const [lectureDoc] = usePrefetchedDynamicDoc(CollectionOf<Lecture>(`courses/${courseName}/lectures`), lectureId, undefined) || [];
```

Comment:
I think that I understand more clearly why did it failed, it was because the `lectureDoc` was undefined. It great that you fixed it using the `fail safe default` principle. Good job 👍.

Reply:
Yup, it was one of the factor for the render to not render everything. It was just a safe feature that I implemented because you already have one protector, which is the checking if lecture is undefined or not just below the code. Nevertheless, having one more safety feature helps our code to be more sure, less prone of errors.

Reply:
I could not have stated it in a more concise fashion that you did 🤐.